### ISSUE TITLE
[Bug]: How to set the boundary of the line object as the body just , because sometimes the line has rotation and the boundary still has rectangle shape.

### ISSUE THREAD

**Issue by design2023** [fabricjs/fabric.js#9719]
### CheckList

- [X] I agree to follow this project's Code of Conduct
- [X] I have read and followed the [Contributing Guide](https://github.com/fabricjs/fabric.js/blob/master/CONTRIBUTING.md)
- [X] I have read and followed the [Issue Tracker Guide](https://github.com/fabricjs/fabric.js/blob/master/CONTRIBUTING.md#%EF%B8%8F-issue-tracker)
- [X] I have searched and referenced existing issues and discussions
- [X] I am filing a **BUG** report.
- [X] I have managed to reproduce the bug **after** upgrading to the **latest** version
- [X] I have created an accurate and minimal reproduction

### Version

6.0.0-beta9

### In What environments are you experiencing the problem?

Chrome

### Node Version (if applicable)

21.6.2

### Link To Reproduction

No reo because the question is simple.

### Steps To Reproduce

1. set the boundary to the same line shape.
2. 
3.

### Expected Behavior

I want to get the boundary to the same line shape.

### Actual Behavior

I want to get the boundary to the same line shape.

**Comment by asturur** [comment]
Hi @design2023 , the only way you have to fix that is to rotate the points before creating the lines.
How are you creating your lines?
Or even better, what are you doing with lines? what functionality do you plan to have with lines?

**Comment by design2023** [comment]
Hi @asturur , thank you for your answer.

I created it by line class from fabric and tried to change the rotation angle of the line, but the boundary still appears like the image, when the line is vertical or horizontal the boundary appears like the line, not more area, I want to make the rotation on boundary also and make it like the size line.

**Comment by design2023** [comment]
> Hi @design2023 , the only way you have to fix that is to rotate the points before creating the lines. How are you creating your lines? Or even better, what are you doing with lines? what functionality do you plan to have with lines?

Hi @asturur, Do you understand what I mean?

**Comment by asturur** [comment]
Yes, i wanted to make an example to guide you but then since it was a bit complicated i digressed. I'll try to do today

**Comment by ShaMan123** [comment]
I override the render or _render method, increase the strokeWidth, change the stroke and call the method before the super execution

**Comment by asturur** [comment]
i just wanted to show how to create a line that is always aligned with the bounding box.
Which version of fabric are you using?

**Comment by asturur** [comment]
@design2023 
https://jsfiddle.net/asturur/6oLsc45v/23/

this is the example i could sketch.

I m not sure what is your exact goal but this is a way to get rid of the box

**Comment by design2023** [comment]
Thank you so much @asturur .

**Comment by NonStopTreasure** [comment]
> Thank you so much @asturur .

HI, sorry for disturb,but i have a question about this.
Does this possible do for polygon. I mean, can bound rect be the same with stroke line that drawn between points ?
I wanna detect intesection of polygon but method detect by bound of objects.
Mb im havent enough read doc and lost it.
Is this possible ?
